From 7114c9a568095fab91cea52d00c6368360c12fb0 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 27 Mar 2013 14:03:50 +0800
Subject: [PATCH 182/281] ENGR00255526 software keyboard is unavailable after
 hotplug/unplug USB keyboard

EventHub does not support hotplug buildin keyboard. It cannot update android
config for buildin keyboard remove event.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 common/input/Dell_Dell_USB_Keyboard.idc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/input/Dell_Dell_USB_Keyboard.idc b/common/input/Dell_Dell_USB_Keyboard.idc
index 2534919..02cbfc4 100644
--- a/common/input/Dell_Dell_USB_Keyboard.idc
+++ b/common/input/Dell_Dell_USB_Keyboard.idc
@@ -19,7 +19,7 @@ touch.orientationAware = 1
 keyboard.layout = Dell_Dell_USB_Keyboard
 keyboard.characterMap = Generic
 keyboard.orientationAware = 1
-keyboard.builtIn = 1
+keyboard.builtIn = 0
 
 cursor.mode = navigation
 cursor.orientationAware = 1
-- 
1.8.0

